doc/common-illnesses.rst

```diff
@@ -0,0 +1,170 @@
+################
+Common Illnesses
+################
+
+Beyond the general troubleshooting steps, some frequently-reported problems require more specialized solutions.
+
+
+
+==========================
+Errors starting the kernel
+==========================
+
+If you receive the message ``An error occurred while starting the kernel`` in the :doc:`ipythonconsole`, Spyder was unable to launch a new Python interpreter in the current working environment to run your code.
+There are a number of problems that can cause this, but most can be fixed fairly quickly with a few easy steps.
+
+
+Spyder-Kernels not installed/incompatible
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Spyder requires a supported version of the ``spyder-kernels`` package to be present in the working environment you want to run your console in.
+
+.. image:: images/common-illnesses/common-illnesses-kernel-version.png
+   :alt: Kernel version error dialog
+
+It is included by default with Anaconda, but if you want to run your code in another Python environment or installation, you'll need to make sure it's installed and updated to the latest version.
+
+Check the required version of spyder-kernels for your version of Spyder in the following table:
+
+.. table:: Spyder and Spyder-Kernels version compatibility
+
+   ==============   ==============
+   Spyder           Spyder-Kernels
+   ==============   ==============
+   4.0.0-4.0.1      1.8.1
```

Comment:
Yeah, but on the other hand its the docs for Spyder 4. Up to @juanis2112 .

If we do implement this, because users are putting this value directly into the conda command, it needs to be a valid version spec and should be the latest version, since it is hardcoded down to the patch. So:

```suggestion
   3.3.0-3.3.6      0.5.2
   4.0.0-4.0.1      1.8.1
```

Reply:
> I think we shouldn't talk about Spyder 3 at all.

Ok, I agree with you @juanis2112. Besides, @CAM-Gerlach's suggestion is not applicable because there are also specific versions of spyder-kernels 0.x that work with specific ones of Spyder (now that I remember).